.drone.yml

```diff
@@ -1,19 +1,35 @@
 ---
 kind: pipeline
 type: docker
-name: linux-amd64
+name: pcgc
 
 platform:
   arch: amd64
   os: linux
 
 steps:
+  - name: test
```

Comment:
a bit of a split, should improve the speed a little, should also help to no longer install `golangci` as they provide a docker image we can use here `golangci`
I've been doing something similar for my project